pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -503,6 +503,420 @@ var _ = Describe("v1beta1 inference service controller", func() {
 			}
 			Expect(actualHPA.Spec).To(gomega.Equal(expectedHPA.Spec))
 		})
+		It("Should have ingress/service/deployment/hpa created with DeploymentStrategy", func() {
+			By("By creating a new InferenceService with DeploymentStrategy in PredictorSpec")
```

Comment:
can't be this test broken into smaller `By` steps?

Reply:
Nope. It would require refactoring existing tests